models/demos/falcon7b_common/tt/falcon_attention.py

```diff
@@ -617,8 +617,9 @@ def forward(
         # key and value layers will have kv_seq_len padded to nearest 32
         key_layer = ttnn.slice(
             layer_past[0],
-            [0, 0, 0, 0],
-            [batch, 1, nearest_32(layer_past_len + 1), self.head_dim],
+            starts=(0, 0, 0, 0),
+            ends=(batch, 1, nearest_32(layer_past_len + 1), self.head_dim),
+            steps=(1, 1, 1, 1),
```

Comment:
1. Can we set steps as default?
2. Is there a possibility to use similar slice mechanism as torch? tensor[dim0_start:dim0_end:dim0_step,...]

Reply:
1. we have a version with nullopt steps too but this is a bit faster since we have fixed sized arrays
2. yes that's already supported!